src/org/thoughtcrime/securesms/WebViewActivity.java

```diff
@@ -333,26 +312,6 @@ protected WebResourceResponse interceptRequest(String url) {
 
   @Override
   public void onActivityResult(int reqCode, int resultCode, final Intent data) {
-    if (reqCode == REQUEST_CODE_FILE_PICKER && filePathCallback != null) {
-      Uri[] dataUris = null;
-      if (resultCode == Activity.RESULT_OK && data != null) {
-        try {
-          if (data.getDataString() != null) {
-            dataUris = new Uri[]{Uri.parse(data.getDataString())};
-          } else if (data.getClipData() != null) {
-            final int numSelectedFiles = data.getClipData().getItemCount();
-            dataUris = new Uri[numSelectedFiles];
-            for (int i = 0; i < numSelectedFiles; i++) {
-              dataUris[i] = data.getClipData().getItemAt(i).getUri();
-            }
-          }
-        } catch (Exception e) {
-          e.printStackTrace();
-        }
-      }
-      filePathCallback.onReceiveValue(dataUris);
-      filePathCallback = null;
-    }
     super.onActivityResult(reqCode, resultCode, data);
```

Comment:
i assume, the whole `@Override onActivityResult()` can be removed here as it does nothing else than calling `super.onActivityResult()`

Reply:
yes, I was thinking that as well, but since it was like that even before adding this code, I thought it was intentional, will remove it then